src/components/Projects/ActionablePersonnelView.tsx

```diff
@@ -70,7 +71,7 @@ const PERSONNEL_INITIAL_VALUES = {
   relevantLink: '',
 } as Partial<PersonnelAttributes>
 
-function ActionablePersonnelView({ members, projectId, isEditor }: Props) {
+function ActionablePersonnelView({ members, projectId, isEditor, isEditable = true }: Props) {
```

Comment:
I'm thinking that if we are always using this two props together on this components, maybe the can receive a `canEdit={isEditor && isEditable}` prop instead